README.md

```diff
@@ -1,7 +1,7 @@
 
-# TDM Reservation Protocol Community Group
+# Text and Data Mining Reservation Protocol Community Group
 
-## Text and Data Mining (TDM) Techniques
+##  Text and Data Mining (TDM) Techniques
```

Comment:
is this double-spacing significant in markdown or some convention above markdown?

Reply:
I have nothing against clarifying the title, but doing so here, isn't it changing the name of the CG itself? see https://www.w3.org/community/tdmrep/

Re the added space, I don't think it has any markdown meaning.

Reply:
> I have nothing against clarifying the title, but doing so here, isn't it changing the name of the CG itself? see https://www.w3.org/community/tdmrep/

We can get around it if we start the CG description text by unfolding the abbreviation. I.e,

> The goal of this Group is to facilitate Text and Data Mining (TDM) in Europe and elsewhere, by specifying a simple and practical machine-readable solution, capable of expressing the reservation of TDM rights - following the rules set by the new European DSM Directive / Art.4 - and the availability of machine-readable licenses for TDM actors.

But we can also try to change the title

> Text and Data Mining (TDM) Reservation Protocol Community Group

I do not think you can do this, but I can try to get it done if you agree.

Is it o.k. to merge this?

Reply:
I'm ok